PhpcrMigration/Infrastructure/Repository/EntityRepository.php

```diff
@@ -12,10 +12,18 @@
 namespace Sulu\Bundle\PhpcrMigrationBundle\PhpcrMigration\Infrastructure\Repository;
 
 use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Exception;
+use Doctrine\DBAL\Schema\Sequence;
 use Sulu\Bundle\PhpcrMigrationBundle\PhpcrMigration\Application\Repository\EntityRepositoryInterface;
+use Symfony\Contracts\Service\ResetInterface;
 
-class EntityRepository implements EntityRepositoryInterface
+class EntityRepository implements EntityRepositoryInterface, ResetInterface
 {
+    /**
+     * @var Sequence[]
+     */
+    private array $sequences;
```

Comment:
Not set in constructor so can be null

```suggestion
    /**
     * @var Sequence[]|null
     */
    private array $sequences;
```